test/e2e/network/dns_common.go

```diff
@@ -269,6 +264,7 @@ func (t *dnsTestCommon) deleteCoreDNSPods() {
 	options := metav1.ListOptions{LabelSelector: label.String()}
 
 	pods, err := t.f.ClientSet.CoreV1().Pods("kube-system").List(options)
+	framework.ExpectNoError(err, "failed to list pod of kube-system with k8s-app = kube-dns")
```

Comment:
list pod -> list pods

Reply:
@neolit123  
well,i will update it.Thanks